.github/workflows/test-pr.yml

```diff
@@ -0,0 +1,35 @@
+name: Test GH action
+
+on:
+  # Scan changed files in PRs, block on new issues only (existing issues ignored)
+  push:
+    branches:
+    - github_action_authentication
+  workflow_dispatch:
+
+env:
+  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+jobs:
+  testauth:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      # Fetch project source
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+      - name: yarn install and build
+        run: |
+          npm install --global cross-env & \
+          yarn install & \
+          yarn run build
+      - name: try the build
+        run: |
+          cat packages/sarif-to-markdown/test/snapshots/suppressed-finding/input.json & \
+          ./packages/sarif-to-comment/bin/cmd.js packages/sarif-to-markdown/test/snapshots/suppressed-finding/input.json \
+          --commentUrl https://github.com/Gby56/security-alert/issues/1 \
```

Comment:
I want to move this test CI to this repository.
I've invited you to this repository.
Free feel to use this repository for testing.